### ISSUE TITLE
docker container

### ISSUE THREAD

**Issue by owocki** [owocki/pytrader#27]
it seems a lot of folks are having a hard time getting the repo up and running (dependancy issues, gettting postgres importing, etc) i was thinking of just creating a VM people can use so that we dont waste more time on installation of libraries.

**Comment by massenz** [comment]
If you don't have objections, I'm happy to have a go at creating the `Dockerfile` and whatever else scripts are necessary (`entrypoint` etc.) - it would be a good way for me to familiarize myself with the structure of the project (and I would do it anyway for myself).

If you are happy with me doing this, please assign this issue to me.
Thanks.

**Comment by pedrommone** [comment]
I'll be happy to help too.

**Comment by owocki** [comment]
@pedrommone @massenz 

According to https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/

> You can only create assignments for yourself, collaborators on personal projects, or members of your organization with read permissions on the repository.

so I don't seem to be able to assign this issue you to.  I'm not sure what the best way to move forward is here.  (1) you can submit a marginal PR (say, moving around a space in the readme) to the repo, which I will accept and which willl make you a collaborator or (2) we can just pretend that this issue is assigned to you.

Also, feel free to sign up for the pytrader slack https://github.com/owocki/pytrader/issues/23 -- I'd be curious to see your guys' notes as you spin up the docker repo.

**Comment by t0mk** [comment]
@massenz there's this: https://github.com/owocki/pytrader/pull/25

**Comment by massenz** [comment]
#25 is sweet, I'll check it out on my system.

@owocki, that's cool, I can do that, no prob.
Just FYI you can also add "collaborators" in the repo Settings (and give various permissions).

**Comment by t0mk** [comment]
#25 merged. I guess I can close this. Deployment to public cloud discussed in #51.
